sdk/storage/storage-blob-changefeed/src/models/BlobChangeFeedEvent.ts

```diff
@@ -0,0 +1,32 @@
+export type BlobChangeFeedEventType = "BlobCreate" | "BlobDeleted";
```

Comment:
what about BlobPropertiesUpdated and BlobSnapshotCreated?

Reply:
What's your reference?
We don't find it here: https://docs.microsoft.com/en-us/azure/event-grid/event-schema-blob-storage?toc=%2fazure%2fstorage%2fblobs%2ftoc.json#tutorials-and-how-tos

Reply:
According to the change feed doc, https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-change-feed?tabs=azure-portal#change-event-records
@seanmcc-msft  do you know whether we need to support BlobPropertiesUpdated and
BlobSnapshotCreated?

>The following event types are captured in the Change feed records:
BlobCreated
BlobDeleted
BlobPropertiesUpdated
BlobSnapshotCreated

Reply:
Confirmed with feature team, more event types need to be supported.
Changed.